packages/core/package.json

```diff
@@ -68,6 +68,7 @@
     "vite": "^6.0.7"
   },
   "dependencies": {
+    "@carbon/utilities": "^0.4.0",
```

Comment:
Since this is being used by the actual `NotificationsPanel` component, we would need to move this into the `packages/ibm-products/package.json`.

However, re-reading the original issue, it looks like we actually intended for the user to use `dateTimeFormat`, which means we would probably want to use the utility in the data or story rather than in the component itself.

Reply:
Hi @elycheea, based on recent meetings, is this still the case?

Reply:
Yep. Move it to our `ibm-products/package.json`!

Reply:
Done.